seahorn/include/proof_allocators.h

```diff
@@ -34,3 +36,6 @@ void *can_fail_malloc(size_t size);
  * Pointer to SeaHorn-based allocator
  */
 struct aws_allocator *sea_allocator(void);
+
+/* allocator where malloc, free and calloc can non-det fail (return NULL)*/
+struct aws_allocator *sea_can_fail_allocator(void);
```

Comment:
`sea_allocator` uses `bounded_malloc` which internally uses `sea_malloc_havoc_safe`; this is not in line with the allocator that CBMC uses for hash_table initialization. I think we should keep `sea_can_fail_allocator` and just set its `mem_calloc` to NULL

Reply:
try switching sea_allocator to `can_fail_malloc`. We should only be using non-failing allocators in our harnesses. That is, where failing allocation is known to be non-interesting behaviour. The only reason for not using failing allocator is that it may increase verification time